go.mod

```diff
@@ -50,8 +50,8 @@ require (
 	github.com/multiformats/go-multiaddr v0.12.2
 	github.com/multiformats/go-multiaddr-dns v0.3.1
 	github.com/multiformats/go-multihash v0.2.3
-	github.com/onflow/atree v0.6.0
-	github.com/onflow/cadence v0.42.10
+	github.com/onflow/atree v0.6.1-0.20240405171204-c6c951de7189
+	github.com/onflow/cadence v0.42.10-0.20240409201024-d000241476d6
```

Comment:
I tagged `v0.42.11-atree-register-inlining` in the Cadence repo, could you please update to that one? 
It is `0.42.10` + atree register inlining (.11 to make the version "newer" than v0.42.10)

Reply:
Thanks for tagging!

Updated this PR to use it in commit [5c24862](https://github.com/onflow/flow-go/pull/5736/commits/5c24862d69bc0ef2c22c5175ae0438f82677210a).